### DISCUSSION TITLE
Jekyll blog build action failed with exit code 1

### DISCUSSION THREAD

**Discussion by PinYuanChen** [community/community#42649] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

Hello, I am a jekyll newbie and I used Chirpy as the template for my blog. A few days ago I could still build my blog without problem. But today I failed with error message as below

```
Error: Error: The process '/opt/hostedtoolcache/Ruby/3.2.0/x64/bin/bundle' failed with exit code 1
    at ExecState._setResult (/home/runner/work/_actions/ruby/setup-ruby/v1/dist/index.js:5371:25)
    at ExecState.CheckComplete (/home/runner/work/_actions/ruby/setup-ruby/v1/dist/index.js:5354:18)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/ruby/setup-ruby/v1/dist/index.js:5248:27)
    at ChildProcess.emit (node:events:390:28)
    at maybeClose (node:internal/child_process:1064:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)
```

The link is [here](https://github.com/PinYuanChen/pinyuanchen.github.io/actions). Thanks for help.

**Comment by DSDmark** [comment, marked as answer]
Hi @PinYuanChen,

Make sure that you are using a compatible version of Ruby and Jekyll, you have all the required gems installed and that they are compatible too. `_config.yml` file is correctly configured and there are no syntax errors check this too.

**Comment by jfullstackdev** [comment]
seems that's compatibility issues

![image](https://user-images.githubusercontent.com/47092464/209561960-5b7e053d-4b23-43b4-87ba-ace6e3a18c33.png)

that can really break your app

**Comment by PinYuanChen** [comment]
It seems like the original script had a problem at setup Ruby stage. I changed the ruby version from 3 to 3.1 and it ran successfully again. Thanks for you guys.

**Reply by chpollin** [reply]
It is in the pages-deploy.yml. It took me a bit to find it. ^^
Thx! Worked for me! :)

**Reply by annavalin** [reply]
Worked for me as well :)
.github/workflows/pages-deploy.yml under name: Setup Ruby
Thxs !!

**Comment by longshilin** [comment]
delete project Gemfile.lock file and github workflow Caches, then work for me.
